lib/erlef/data/schema/academic_paper.ex

```diff
@@ -0,0 +1,60 @@
+defmodule Erlef.Data.Schema.AcademicPaper do
+  @moduledoc """
+  Schema module for academic papers
+  """
+  use Erlef.Data.Schema
+
+  @type t :: %__MODULE__{
+          id: binary(),
+          title: String.t(),
+          author: String.t(),
+          language: String.t(),
+          url: String.t(),
+          pay_wall?: :boolean | nil,
+          description: String.t() | nil,
+          technologies: {:array, String.t()} | nil,
+          original_publish_date: Date.t() | nil,
+          published_at: :utc_datetime | nil,
+          deleted_at: :utc_datetime | nil
+        }
+
+  @required_fields ~w|
```

Comment:
This is a pattern I've personally gotten away from and now just do `[:atom, :atom, :atom, :atom]` since mix format can properly format the latter.

Reply:
habits, we all have them :)